15-7-1_and_15-c-1/update_15-7-1_and_15-c-1.R

```diff
@@ -0,0 +1,90 @@
+
+# Code to automate data update for indicator 15-7-1 and 15-c-1 (number of endangered species seizures)
+
+# read in data 
+
+seizures_source_data <- get_type1_data(header_row, filename, tabname)
+
+# remove cells above column names
+
+seizures_main_data <- extract_data(seizures_source_data, header_row)
+
+# remove superscripts from column names
+
+names(seizures_main_data) <- SDGupdater::remove_superscripts(names(seizures_main_data)) 
+
+# rename columns
+
+renamed_seizures_main_data <- seizures_main_data %>% 
+  rename_column(primary = "Total Number of Seizures", new_name = "Total Number of Seizures") %>% 
+  rename_column(primary = "Number of Times seized: Caviar & Caviar extract", new_name = "Caviar and Caviar extract") %>% 
+  rename_column(primary = "Number of Times seized: Live Coral & Coral Derivatives", new_name = "Live Coral and Coral Derivatives") %>% 
+  rename_column(primary = "Number of Times seized: Ivory and Items Containing Ivory", new_name = "Ivory and Items Containing Ivory") %>% 
+  rename_column(primary = "Number of Times seized: Live Animals and Birds", new_name = "Live Animals and Birds") %>%
+  rename_column(primary = "Number of Times seized: Live Plants", new_name = "Live Plants") %>%
+  rename_column(primary = "Number of Times seized: Parts or Derivatives of Animals/Birds", new_name = "Parts or Derivatives of Animals or Birds") %>%
+  rename_column(primary = "Number of Times seized: Parts or Derivatives of Plants", new_name = "Parts or Derivatives of Plants") %>%
+  rename_column(primary = "Number of Times seized: Timber or Wood Products", new_name = "Timber or Wood Products") %>%
+  rename_column(primary = "Number of Times seized: Preparations Of Oriental Medicine Which Include Parts & Derivatives Of Endangered Species", new_name = "Preparations Of Oriental Medicine Which Include Parts and Derivatives Of Endangered Species") %>%
+  rename_column(primary = "Number of Times seized: Butterflies", new_name = "Butterflies") 
+
+# split year and quarter column into two
+
+renamed_seizures_main_data[c('Year', 'Quarter')] <- str_split_fixed(renamed_seizures_main_data$Quarter, ' ', 2)
+
+# select wanted columns
+
+renamed_seizures_main_data <- renamed_seizures_main_data %>% 
+  select(c("Year", "Quarter", "Total Number of Seizures", "Caviar and Caviar extract", "Live Coral and Coral Derivatives", "Ivory and Items Containing Ivory", "Live Animals and Birds", "Live Plants", "Parts or Derivatives of Animals or Birds", "Parts or Derivatives of Plants", "Timber or Wood Products", "Preparations Of Oriental Medicine Which Include Parts and Derivatives Of Endangered Species", "Butterflies"))
+
+# remove rows containing only NAs
+
+renamed_seizures_main_data <- na.omit(renamed_seizures_main_data)
+
+# change ":" to NAs
+
+renamed_seizures_main_data[renamed_seizures_main_data == ":"] <- "NA" 
+
+# calculate total seizures for each import type for each year
+
+seizures_main_data_totals <-  renamed_seizures_main_data %>% 
```

Comment:
Looks like here you're using this to just drop the Quarters column, so it'd be more straight forward to tell it what to stop, than what to select. For example replace code at line 52 with:
`seizures_main_data_totals <- subset(renamed_seizures_main_data, select = -Quarter)`

Reply:
Done